frontend/css/fresh-releases.less

```diff
@@ -19,13 +19,22 @@
   justify-content: space-between;
   padding-bottom: 0.5rem;
   margin-bottom: 1rem;
+
+  @media screen and (max-width: @screen-md) {
+    justify-content: center;
+  }
 }
 
 .fresh-releases-row {
   gap: 0.5rem;
   display: flex;
   justify-content: center;
   align-items: center;
+
+  #user-releases:disabled {
```

Comment:
I think it could be useful in other cases to have a "disabled" style for the pill, buttons.
That would mean moving these css definitions to the pill.less file.

I would also recommend doing like Bootstrap and defining both the disabled attribute and a .disabled css selector, so that someone in the future can use one or the other without requiring specific knowledge.

Reply:
Fixed